vendor/github.com/giantswarm/backoff/backoff.go

```diff
@@ -5,7 +5,7 @@ import (
 )
 
 const (
-	LongMaxWait   = 40 * time.Minute
+	LongMaxWait   = 80 * time.Minute
```

Comment:
There's no changes for `Gopkg.toml` or `Gopkg.lock` in this PR. Is this maybe some kind of leftover when the only other change is removal of `nodename_file_optional`?

Reply:
This was done for testing only, i'm not going to merge this.

We discussing here https://gigantic.slack.com/archives/C2MS2VB37/p1547199975002200